easybuild/easyblocks/c/cplex.py

```diff
@@ -110,11 +110,21 @@ def post_install_step(self):
             raise EasyBuildError("No bins found using %s in %s", binglob, self.installdir)
 
     def make_module_extra(self):
-        """Add installdir to path and set CPLEX_HOME"""
+        """Add bin dirs and lib dirs and set CPLEX_HOME and CPLEXDIR"""
+
+        os.chdir(self.installdir)
```

Comment:
why not use `os.path.join(self.installdir, '*', 'bin', 'x86-64*')` instead of changing to this directory?

if you insist on changing there, make sure you change back to where you were?